### ISSUE TITLE
AstroAuth (Receiving Undefined Options)

### ISSUE THREAD

**Issue by wilsonsergio2500** [nowaythatworked/auth-astro#56]
I am attempting to introduce auth0 as a provider in my project. Hence my auth.config.ts is as such

```js
import Auth0 from "@auth/core/providers/auth0";
...
export default {
  providers: [
    Auth0({
      clientId: [value],
      clientSecret: [value],
    }),
  ],
}
```

Upon login in my user, I am receiving the error *Cannot read properties of undefined (reading 'secret')*
The challenge is that the API call which calls AstroAuth is receiving an empty (or undefined) authConfig options.
Upon following the configuration convention in the readme, I am not certain why this method is not receiving the values introduce in the auth.config.ts declaration. At least this what it looks like by the type (what shall be received by this method)

Any guidance, direction, recommendation or clues will be very much appreciate it.

**Comment by nowaythatworked** [comment]
Hi, did u [setup your environment variables](https://github.com/nowaythatworked/auth-astro#setup-environment-variables) correctly?

**Comment by wilsonsergio2500** [comment]
@nowaythatworked thank you very much for you response. I did.

and importantly this values are coming thru and accessible via *import.meta.env*

![image](https://github.com/nowaythatworked/auth-astro/assets/2171533/d179a75e-d0d8-46f5-8cff-c3f41e85fcfa)

but it blows in line 86 as *options* is undefined... ponder where is my missing.

**Comment by wilsonsergio2500** [comment]
@nowaythatworked I apologize for the false signal here.

After a few days of struggle I attempted the oldest trick in book - start new or fresh. Once I started fresh in a brand new project the library and integration worked on the first try. Not sure what was the challenge before given that the one prior was also a light and fresh boilerplate. 

Once again, I apologize and keep up the good work 👍
